physical_properties/pKa/experimental_data/purity/README.md

```diff
@@ -0,0 +1,23 @@
+## Purity of SAMPL6 pKa challenge solid compound stocks
+
+LC-MS based purity measurements were performed by Heather Wang (Merck).
+
+
+### LC-MS Method 
+The purity analysis was performed using an Agilent HPLC1200 Series equipped with auto-sampler, UV diode array detector and a Quadrupole MS detector 6140. Software was used is Chemistation for LC & LC/MS with version C01.07SR2.    
```

Comment:
"Software was used is Chemistation..." 
to
"The software used is Chemistation..." 
That still sounds a little awkward, but I couldn't come up with something else